cirq-google/cirq_google/serialization/arg_func_langs.py

```diff
@@ -291,6 +293,8 @@ def arg_from_proto(
             return [str(v) for v in arg_value.string_values.values]
         if which_val == 'value_with_unit':
             return tunits.Value.from_proto(arg_value.value_with_unit)
+        if which_val == 'bytes_value':
+            return bytes(arg_value.bytes_value)
```

Comment:
Note (no change necessary) - perhaps these iffs could be converted to a match-case block.

Reply:
SGTM, maybe in a follow-up PR.